core/domain/question_services_test.py

```diff
@@ -168,8 +168,9 @@ def test_get_question_skill_links_of_skill(self):
         self.save_new_question(
             question_id_3, self.editor_id,
             self._create_valid_question_data('ABC'))
+        # Setting skill difficulty for self.question_id.
         question_services.create_new_question_skill_link(
-            self.question_id, 'skill_1')
+            self.question_id, 'skill_1', skill_difficulty=0.5)
```

Comment:
Drop `skill_difficulty=` part, here and elsewhere, if we are getting rid of the default value.

Reply:
Done